woocommerce/wc-admin
# Add filter variation to tracks data in products analytics

Fixes #6221

The filter variation type (`single_variation` or `compare-variations`) is missing from the recorded tracks event. This PR adds it as `filter_variation`.

### Detailed test instructions:

- Have a product with several variations
- Open Analytics -> Products
- Show "SIngle product" and select your variable product
- In the console there should be an event like this: `{report: "products", filter: "single_product", filter_variation: "all"}`
- Select "Single variation" and select a variation
- In the console there should be an event like this: `{report: "products", filter: "single_product", filter_variation: "single_variation"}`
- Select "Comparison"
- In the console there should be an event like this: `{report: "products", filter: "single_product", filter_variation: "compare-variations"}`
